morph/src/Status.re

```diff
@@ -0,0 +1,2 @@
+external to_httpaf: Morph_core.Status.t => Httpaf.Status.t = "%identity";
```

Comment:
We should see if we can do something better here but this should be safe as I think our `Status.t` is a safe subset of both.